src/components/fee-summary/FeeSummary.vue

```diff
@@ -63,9 +63,9 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Emit, Prop } from 'vue-property-decorator'
-import { FeeSummaryActions } from '@bcrs-shared-components/enums'
-import { FilingDataIF } from '@bcrs-shared-components/interfaces'
-import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import { FeeSummaryActions } from '../../enums'
+import { FilingDataIF } from '../../interfaces'
+import SbcFeeSummary from './SbcFeeSummary.vue'
```

Comment:
These really _should_ import the shared items.

This might work here (in SB) but probably won't work when imported into the host app.